CI on own fork always fails because the branch is out of date albeit it isn't

### Is this a new report?

Yes

### System Info

Void 6.1.1_1 x86_64 AuthenticAMD notuptodate hold rrrmDFFFFFFFFFFF

### Package(s) Affected

no specific package

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

https://github.com/steinex/void-packages/pull/230
https://github.com/steinex/void-packages/pull/227

### Expected behaviour

CI should run as usual.

### Actual behaviour

To check if my package builds for all archs, I always open a PR on my own branch first for QA. This stopped working with the problem:
```
Error: Your branch is too out of date. Please rebase on upstream and force-push.
Error: Process completed with exit code 1.
```

This does only happen on my own branch, if I push directly to void-packages everything is fine.


### Steps to reproduce

1. Prepare a PR
2. Push to own branch
3. See CI failing

--- a/common/travis/changed_templates.sh
+++ b/common/travis/changed_templates.sh
@@ -10,7 +10,7 @@ case "$tip" in
 	*)         tip="${tip%% *}" ;;
 esac
 
-base="$(git merge-base FETCH_HEAD "$tip")"
+base="$(git merge-base origin/HEAD "$tip")"
 
 [ $(git rev-list --count "$tip" "^$base") -lt 200 ] || {
 	echo "::error title=Branch out of date::Your branch is too out of date. Please rebase on upstream and force-push."

